client/src/DeleteButton.stories.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react'
+import DeleteButton from './DeleteButton'
+import { MemoryRouter as Router } from 'react-router-dom'
+
+export default {
+  title: 'DeleteButton',
+  component: DeleteButton,
+}
+
+export const simple = () => {
+  return (
+    <Router>
```

Comment:
den Router könntest du auch in die preview.js packen, dann musst den nicht immer extra reinschreiben. Hab ich gestern bei  mir gemacht, kann ich dir gerne mal zeigen

Reply:
Da komm ich gerne nochmal drauf zurück. Ich muss storiemäßig wirklich mal ran.

Reply:
Storiemäßig muss ich einiges aufarbeiten. Komme garantiert nochmal auf dein Angebot zurück ;)